libraries/ESP8266WiFi/src/ESP8266WiFiAP.cpp

```diff
@@ -333,3 +333,33 @@ String ESP8266WiFiAPClass::softAPmacAddress(void) {
     sprintf(macStr, "%02X:%02X:%02X:%02X:%02X:%02X", mac[0], mac[1], mac[2], mac[3], mac[4], mac[5]);
     return String(macStr);
 }
+
+/**
+* Get the configured(Not-In-Flash) softAP SSID name.
+* @return String SSID.
+*/
+String ESP8266WiFiAPClass::softAPSSID() const {
+	struct softap_config config;
+	wifi_softap_get_config(&config);
+	char* name = reinterpret_cast<char*>(config.ssid);
```

Comment:
why not just 
```
config.ssid[sizeof(config.ssid) - 1] = 0; // guard
return String((const char*)(void*)config.ssid) ?
```
String(char*) will anyway search for the last \0

Reply:
@d-a-v 
Based on @devyte  comment user must be able to configure `64` byte length `PSK` at max, not `63`

Reply:
OK sorry I did not see the hidden-outdated comment.